hrjobcontract/js/src/job-contract/services/utils.js

```diff
@@ -118,7 +118,8 @@ define([
                 var deffered = $q.defer();
 
                 API.resource('HRPayScale','get',{
-                    sequential: 1
+                    sequential: 1,
+                    is_active: 1
```

Comment:
The thing you can test is simply: does `getPayScaleGrade` call the `API.resource` service with the following parameter: `'HrPayScale', 'get', ' { sequential: 1, is_active: 1}`?

You will mock the 'API' service to do that, so no ajax request will be made:
```js
spyOn(API, 'resource');
// ...
expect(API.resource).toHaveBeenCalledWith(/* list of param */);
```